tc/src/main/java/com/epam/tc/security/WebSecurityConfig.java

```diff
@@ -29,7 +30,8 @@ public void registerGlobalAuthentication(AuthenticationManagerBuilder auth) thro
     protected void configure(HttpSecurity http) throws Exception {
         http
                 .authorizeRequests()
-                .antMatchers("/login").permitAll()
+                .antMatchers("/login", "/courses", "/courses/{id}/subscribe").permitAll()
+                .antMatchers("/courses/create", "/courses/**/update").hasAuthority("Lector")
```

Comment:
Is this possible to use {courseId} instead of ** here (as above)? If yes, please use this.